File: C/3 Seleção/semana.c
#include <stdio.h>

int main(){
    int dia, hora;
    scanf("%d %d", &dia, &hora);

    if(dia==1) printf("NAO\n");
    else if(dia==7){
        if(hora>=8 && hora<=11) printf("SIM\n");
        else printf("NAO\n");
    }

    else{
        if((hora>=8 && hora<=11) || (hora>=14 && hora<=17)) printf("SIM\n");
        else printf("NAO\n");
    }
}

File: Fup is Fun/Python/3 Seleção/semana.py
dia = int(input())
hora = int(input())

if dia == 1: print("NAO")

elif dia == 7:
    if hora>=8 and hora<=11: print("SIM")
    else: print("NAO")
    
else:
    if (hora>=8 and hora<=11) or (hora>=14 and hora<=17): print("SIM")
    else: print("NAO")